pkg/gardenlet/apis/config/types.go

```diff
@@ -109,6 +109,8 @@ type GardenletControllerConfiguration struct {
 	Shoot *ShootControllerConfiguration
 	// ShootCare defines the configuration of the ShootCare controller.
 	ShootCare *ShootCareControllerConfiguration
+	// ShootState defines the configuration of the ShootState controller
+	ShootState *ShootStateControllerConfiguration
```

Comment:
I'd rather have this named `ShootStateSync` as indeed this does not reconcile the ShootState resource: https://github.com/gardener/gardener/pull/1731#discussion_r361968553

Reply:
@swilen-iwanow what about this comment from @plkokanov ?
Also, can you please expose this new config option in the Gardenlet Helm chart as well as in the [example configuration](https://github.com/gardener/gardener/blob/master/example/20-componentconfig-gardenlet.yaml)?